src/index.ts

```diff
@@ -24,6 +24,10 @@ export async function run(): Promise<void> {
     // remove white spaces (if any) in archs input
     archs = archs.replace(/\s+/g, "");
 
+    if (!tagsList.length) {
+        core.info(`Input "tags" is not provided, using default tag "latest"`);
```

Comment:
rather than hard-coding "tags", how about `${Inputs.TAGS}`

Reply:
likewise `latest` could be captured into a constant like `DEFAULT_TAG`

Reply:
> rather than hard-coding "tags", how about `${Inputs.TAGS}`

Then if `${Inputs.TAGS}` is empty, it will be like;
```
Input is not provided, using default tag "latest"
```

Reply:
and what's the point of adding `${INPUTS.TAGS} if we know it's gonna be empty :thinking:

Reply:
I mean use the inputs-outputs Inputs enum. Inputs.Tags is the string "tags", it will never be empty

Reply:
Oops.. my bad, got confused.
Sorry for that